lib/generators/sorcery/install_generator.rb

```diff
@@ -90,6 +90,12 @@ def only_submodules?
         options[:migrations] || options[:only_submodules]
       end
 
+	  def rails_version # for Rails5 ActiveRecord::Migration 
+		if    Rails::VERSION::MAJOR >= 5
+		  "[#{Rails::VERSION::MAJOR}.#{Rails::VERSION::MINOR}]"
+		end
+	  end
+
```

Comment:
Huh, interesting. I haven't had the chance to really play around with the rails 5 changes yet. I'll have to check it out sometime soon.

Thanks for fixing the whitespace. :+1: